integration/consul_catalog_test.go

```diff
@@ -22,16 +22,12 @@ type ConsulCatalogSuite struct {
 }
 
 func (s *ConsulCatalogSuite) SetUpSuite(c *check.C) {
-	project, err := docker.NewProjectFromEnv()
-	c.Assert(err, checker.IsNil, check.Commentf("Error while creating docker project"))
-	s.project = project
+	s.project = docker.NewProjectFromEnv(c)
 
 	s.createComposeProject(c, "consul_catalog")
-	err = s.composeProject.Start()
-	c.Assert(err, checker.IsNil, check.Commentf("Error starting project"))
+	s.composeProject.Start(c)
 
-	consul, err := s.project.Inspect("integration-test-consul_catalog_consul_1")
-	c.Assert(err, checker.IsNil, check.Commentf("Error finding consul container"))
+	consul := s.project.Inspect(c, "integration-test-consul_catalog_consul_1")
```

Comment:
That's a good point, I've just update the code and let the same assumption that it will always be that (because of the name convention of `compose`).

But something like `s.composeProject.InspectContainer("catalog_consul", 1)` or `s.composeProject.Containers("catalog_consul")[1]` would be better I guess :angel:.